aws/resource_aws_codebuild_project_test.go

```diff
@@ -2477,6 +2512,40 @@ resource "aws_codebuild_project" "test" {
 `, rName, gitCloneDepth)
 }
 
+func testAccAWSCodeBuildProjectConfig_Source_GitSubmodulesConfig(rName string, fetchSubmodules bool) string {
+	fetchSubmodulesValue := "false"
+
+	if fetchSubmodules {
+		fetchSubmodulesValue = "true"
+	}
```

Comment:
Nit: This logic can be removed by using the `%t` format verb below for the boolean. 😄

Reply:
Oh much better :) Thanks for pointing it now, first time attempting to use Go, not very familiar with it's version of `sprintf`.